ex03.exs

```diff
@@ -50,12 +50,39 @@ defmodule Ex03 do
      it is nicely structured            7
   """
 
+
+# Parent function for cuncurrently mapping a collection/func
   def pmap(collection, process_count, function) do
-    # your code goes here
-    # I'm hoping to see a simple pipeline as the body of this function...
+    divideChunks(collection, process_count)
+    #Divide chunks
+    |> delegateProcesses(function)
+    #Delegate for chunks
+    |> List.flatten
+    # Move chunks back into a flat list
+  end
+
+# Divides the collection into the specified number of processes
+  def divideChunks(collection, process_count) do
+    collection |> Enum.chunk_every(div(Enum.count(collection), process_count))
```

Comment:
The `div` returns the incorrect value if the collection size is not a multiple of the process count (try 10 and 4).

# -2